src/org/labkey/test/tests/StudyBaseTest.java

```diff
@@ -212,6 +214,11 @@ protected void exportStudy(boolean zipFile, boolean exportPhi, PhiSelectType exp
         waitForText("Visit Map", "Cohort Settings", "QC State Settings", "CRF Datasets", "Assay Datasets",
                 "Dataset Data", "Specimens", "Specimen Settings", "Participant Comment Settings");
```

Comment:
Remove "Specimens" and "Specimen Settings" from the list of text to wait for.

Reply:
Oops